gulp/docs.js

```diff
@@ -171,6 +172,10 @@ module.exports = {
 					}
 
 					var currentBranch = stdoutBranch.trim().replace(/\//g, '-'); // clean branch and replace forward slashes
+					var isTagged = !err && currentBranch === 'master';
```

Comment:
This should fix the issue you were having @sodiumjoe when you were doing docs upload from a branch without a commit.